doc/utilities/cubes.ipynb

```diff
@@ -167,7 +168,8 @@
     "        a list containing lists of satelites idx occupying the same cube\n",
     "    \"\"\"\n",
     "    # init\n",
-    "    retval = []\n",
+    "    collision = []\n",
```

Comment:
I would call this variable `in_same_cube`

Reply:
Done